src/main/java/com/xml/Xml.java

```diff
@@ -0,0 +1,39 @@
+package com.xml;
+
+import java.beans.XMLEncoder;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.logging.Logger;
+
+public class Xml {
+    private static final Logger LOGGER = Logger.getLogger(Xml.class.getName());
+
+    public void makeMultiLayer() {
+        List<Map<List<Set<Integer>>, String>> multiLayer = new ArrayList<>();
+        Integer number = 1;
+        Set<Integer> set = new HashSet<>(number);
+        List<Set<Integer>> list = new ArrayList<>();
+        list.add(set);
+        Map<List<Set<Integer>>, String> map = new HashMap<>();
+        map.put(list, "Test");
+        multiLayer.add(map);
+        makeXmlFile(multiLayer);
+    }
+
+    private void makeXmlFile(List list) {
```

Comment:
Please use finally block of try-with-resources to close all streams.
Look here https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html